Makefile

```diff
@@ -1,6 +1,6 @@
 export LC_ALL = C.UTF-8
 
-ENVIRONMENT ?= regiocloud
+ENVIRONMENT ?= the_environment
```

Comment:
I would prefer to have regiocloud as default here.

Reply:
I presetted that with "the_environment" to improve the usability.

If you call a target which requires the specification of a valid environment, the output of the target looks now like this:
```sh
$ make create
....
you have to specify a name below the 'clouds' element in 'terraform/clouds.yaml', configured environments are: regiocloud, pluscloudopen, gx-scs (set it with ENVIRONMENT=...)
make: *** [Makefile:29: create] Error 1
```

If you specify a undefined environment you receive the following message:
```sh
make ENVIRONMENT=yolobanana create
....
no such key 'clouds.yolobanana' in 'terraform/clouds.yaml', configured environments are: regiocloud, pluscloudopen, gx-scs (set it with ENVIRONMENT=...)
make: *** [Makefile:29: create] Error 1
```